src/main/java/io/thinkit/edc/client/connector/JsonLdUtil.java

```diff
@@ -26,4 +26,10 @@ public static JsonObject compact(JsonLdObject input) throws JsonLdError {
                 .build());
         return JsonLd.compact(expanded, context).get();
     }
+
+    public static ApiErrorDetail getError(InputStream body) throws JsonLdError {
```

Comment:
I won't reference `ApiErrorDetail` in this `JsonLdUtil`, I think it should return the `JsonObject` and then the error object should be instantiated by the caller